scripts/update-firebase-sri.sh

```
@@ -62,8 +65,29 @@ declare -a FILES=(
+  # this script bumped FOUR of the five tags and silently left
+  # firebase-functions-compat.js on the OLD version — a mixed-version compat
+  # load, precisely the breakage this script exists to prevent.
+  "firebase-functions-compat.js"
 )
 
+# Fail rather than half-update. If index.html references a firebasejs file
+# this array does not know about, the run would leave that tag behind on the
+# old version and report success — so refuse instead, naming the offender.
+MISSING=""
+for seen in $(grep -oE 'firebasejs/[0-9]+\.[0-9]+\.[0-9]+/[a-z-]+\.js' "$INDEX_HTML" \
+              | sed 's|.*/||' | sort -u); do
```

Shellcheck (0.11.0)

[info] To read lines rather than words, pipe/redirect to a 'while read' loop.

(SC2013)